src/themes/dark.scss

```diff
@@ -1,3 +1,18 @@
 .wxc-theme-dark {
   --wxc-meeting-background: linear-gradient(180deg, #1A1A1A 0%, #0F0F0F 100%);
+
+  --wxc-remote-media-connecting-spinner-border-color: $gray-60;
+  --wxc-remote-media-connecting-spinner-dimension: 0.875rem;
```

Comment:
please remove all size-related variables (border width, margin, padding, heights, etc.) and just hardcode the values in the component css